code/entities/machine/soda-fountain/SodaFountain.FillableCup.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Sandbox;
+
+namespace Cinema;
+
+public partial class FillableCup : ModelEntity
+{
+    public Dispenser Dispenser { get; set; }
+    public string ItemId { get; set; } 
+    public bool IsAssembled { get; set; }
```

Comment:
Clean Code Practice:

Make this a protected `set` and explicitly default this to `= false`

Reply:
Changed IsAssembled to protected set and set the default to false